### DISCUSSION TITLE
Remove eQSL from Dashboard

### DISCUSSION THREAD

**Discussion by 4O4A** [wavelog/wavelog#297] [Ideas]
Hi guys,

It is possible to remove eQSL column in log if eQSL data is not set in Third Party Services tab.
Is it possible to have eQSL removed from Dashboard, following the same criteria?

Regards,
Dragan, 4O4A

**Comment by int2001** [comment]
it is already removed from dashboard if you have no eQSL-Sent or Received at the active station. 
(Fields in ADIF/Database counts --> if you ever imported ADIFs with "mark eQSL as sent", eQSL at Dashboard will be there)

Regarding the column at Overview - yes. Good idea to hide the Col if no credentials are set.

**Reply by int2001** [reply]
Wait @4O4A - if i remove eQSL-Infos from **Account-Setting** the column disappears.
Or are you talking about sth. different?

**Reply by 4O4A** [reply]
Hm. I removed info about eQSL in Account settings, and that caused that I don't see eQSL column in log, as it should be.
But, eQSL info is still visible in right hand column on Dashboard page.

![image](https://github.com/wavelog/wavelog/assets/101504692/61a68d83-3d9f-4417-b591-bc3eee9a5c95)

Speaking about Dashboard, I reorganized info in my Dashboard (Application/View/Dashboard/index.php) in manner that first I see DXCC, then VUCC, then QSL info -- rather than DXCC, QSL info, VUCC. But, that's easy to reoder, to satisfy anyone's OCD :)

**Reply by int2001** [reply]
The eQSL-Section at Dashboard is shown, because you sent and received eQSLs.
The section disappears if you never synced with eQSL
